.env.example

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_DEV_CLIENT_ID=
+NEXT_PUBLIC_DEV_CLIENT_ID=https://storage.inrupt.com/87048fc7-b553-4c86-95de-633e1675f0bd/clientid.jsonld
```

Comment:
Do we need this value populated in the example env?

Reply:
Normally you'd only have `.env.example` checked in, so it makes sense to have the value there. Though for some reason we check in `.env.development` too.